tests/c_api_tests/test_cpp_api_model.cpp

```diff
@@ -180,7 +194,7 @@ TEST_CASE("C++ API Model") {
     }
 
     SUBCASE("Copy model") {
-        Model model_copy{model};
+        Model const& model_copy{model};
```

Comment:
this results in a reference, not a copy.
```suggestion
        Model model_copy{model};
```

If `clang-tidy` complains, this is where you can use `NOLINT(<identifier>)`

Reply:
Issue already fixed. Comment resolved.